config/delta-producer/organizations/export.json

```diff
@@ -389,7 +389,8 @@
     {
       "type": "http://www.w3.org/ns/prov#Location",
       "graphsFilter": [
-        "http://mu.semte.ch/graphs/shared"
+        "http://mu.semte.ch/graphs/shared",
+        "http://mu.semte.ch/graphs/public"
```

Comment:
We can only keep the public graph now that all the Locations are in there no? I checked and no location is left in the shared graph